application/backend/config/packages/security.yaml

```diff
@@ -50,4 +54,5 @@ security:
     access_control:
         - { path: ^/api/token, roles: IS_AUTHENTICATED_ANONYMOUSLY }
         - { path: ^/api/docs, roles: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/api/dictionaries , roles: IS_AUTHENTICATED_ANONYMOUSLY }
```

Comment:
Visible without auth ? No need to be protected under the main firewall ? No sensitives data in dictionaries ?